src/_array_api_conf.py

```diff
@@ -1,30 +1,25 @@
-# Configuration file for the Sphinx documentation builder.
```

Comment:
Why is this file in `src/`? Shouldn't it be in `spec/`?

Reply:
It gets installed with `pip install .`. Seemed nicer than adding it to `spec/` and making relative imports, given you then you're in the territory of having to treat `spec/` and its sub-folders as Python modules.

Reply:
I find it somewhat confusing to do it this way. You generally don't install Sphinx conf files.

Reply:
Right, but you also generally don't keep contents of multiple doc versions in a repo. Something probably has to give here :sweat_smile: